larray/inout/pandas.py

```diff
@@ -291,9 +293,27 @@ def _series_to_group(series, axis):
     return LGroup(key=series.values, name=name, axis=axis)
 
 
-# ################################## #
-#    DATAFRAME <--> AXES, GROUPS     #
-# ################################## #
+def _metadata_to_series(metadata):
+    s = pd.Series(data=list(metadata.values()), index=metadata.keys(), name='')
```

Comment:
Why don't you use a 1D object LArray for this?

Reply:
Because LArray objects are converted to pandas objects before to be dumped or loaded. I think using 1D array here instead of a pandas Series will add an unnecessary layer. Especially since we need to iterate over each element at reading to convert each metadata to the right type.

Reply:
This might be a case of premature optimization. Passing via LArray has negligible cost in my working copy (disclaimer: I modified .series slightly to go faster for 1D and I had already a few speed optimizations all over the place in my working copy, so this might not be the case on your computer).

```python
>>> metadata = {'title': 'yad aez dgrz jgzeigfejzi', 'date': datetime.now()}
>>> %timeit l = LArray(list(metadata.values()), Axis(metadata.keys())); s = l.series; s.name = ''
1.22 ms ± 57.8 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)
>>> %timeit s = pd.Series(data=list(metadata.values()), index=metadata.keys(), name='')
1.3 ms ± 65.8 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)
```

Besides, I think that it might be a case where avoiding pandas structures altogether would be faster. Going via pandas is a net win for reading .csv files because their parser is so fast, but for anything else, we only gain convenience, not performance. This is probably especially true when we go via xlwings, which in the end must transform everything to "simple" Python types. Anyway, you can leave this as-is, but please at least add a comment that this is done for performance reasons.